### PR TITLE
updated openy_analytics php version extraction method

### PR DESCRIPTION
**Pull request by retif**

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_analytics/src/AnalyticsCron.php [hunk 1]
```diff
@@ -118,7 +118,7 @@ function getServerInfo() {
     $db_detailed_version = $this->database->query("SHOW VARIABLES LIKE '%version%';")
       ->fetchAllKeyed();
     $server_software = $_SERVER['SERVER_SOFTWARE'];
-    $php_version = $_SERVER['PHP_VERSION'];
+    $php_version = phpversion();
```
Comment (podarok) [inline modules/custom/openy_analytics/src/AnalyticsCron.php:121]: Note that the version string returned by phpversion() may include more information than expected: "5.5.9-1ubuntu4.17", for example.
From https://www.php.net/phpversion
Are we fine with having this string as is?

**Review by podarok — APPROVED** [review]
@retif the question

File: modules/custom/openy_analytics/src/AnalyticsCron.php (hunk 1 above)
Comment (podarok) [reply modules/custom/openy_analytics/src/AnalyticsCron.php:121]: I can answer.
Yes, we are good
Thank you )))

**Pull request merged**